core/types.go

```diff
@@ -145,11 +145,12 @@ func (ch *Challenge) ExpectedKeyAuthorization(key *jose.JSONWebKey) string {
 }
 
 type Certificate struct {
-	ID        string
-	Cert      *x509.Certificate
-	DER       []byte
-	Issuer    *Certificate
-	AccountID string
+	ID                 string
+	Cert               *x509.Certificate
+	DER                []byte
+	Issuer             *Certificate
+	AlternativeIssuers []*Certificate
```

Comment:
Ditto my advice here about the chain fields. I think just one field of `[]*Certificate` would be simpler, with the default issuer certificate being index 0.

Reply:
Ah, I overlooked this apparently when I wrote the comment above; this is also changed :)